pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -250,7 +250,7 @@ protected void validateAdminAndClientPermission() {
     protected void validateCreateTopic(TopicName topicName) {
         if (isTransactionInternalName(topicName)) {
```

Comment:
The topic `TRANSACTION_COORDINATOR_ASSIGN` could be created by users, right?

Reply:
yes